config/default.json

```diff
@@ -29,5 +29,12 @@
 		"SERVERNAME": "messenger.schule",
 		"SECRET": "SECRET",
 		"STUDENT_ROOM_CREATION": false
+	},
+	"COOKIE": {
```

Comment:
It is true that embedded object default values in default.schema.json do not work well. One of the reason why we want to replace it with the nestjs solutions. ;)

For fixing it we hold the default.json alive. But the default.json should only used for embedded config object to that the default values for production. 
For development settings we use the development.json.

I think you use it in this way, but i am confusing about the different between default.json values and the default.schema.json values that should represent the production defaults.

Reply:
we actually copied the values from the client.  Secure=false in default.json and secure = true in default.schema.json. If this is not what it should be, it should be changed here and in client.

Reply:
Only the values from default.json and the embedded defaults in default.schema.json should be the same. 

Maybe the first draft can explain why. 
https://github.com/hpi-schul-cloud/schulcloud-server/tree/BC-2347-add-configuration-documentation/config

Reply:
ok, done. The values are the production values from the client.